style/gallery.less

```diff
@@ -214,6 +217,93 @@
 		}
 	}
 
+	&-attribution {
+		border: #f8f9fa;
+		height: 44px;
+		display: flex;
+		justify-content: space-between;
+		margin-bottom: 15px;
+
+		&-info {
+			display: flex;
+			align-items: center;
+			margin-left: 16px;
+
+			&-cc {
+				background-image: inline-svg( '../images/attribution-cc.svg' );
+				background-repeat: no-repeat;
+				width: 14px;
+				height: 14px;
+				margin-right: 8px;
+			}
+
+			&-by {
+				background-image: inline-svg( '../images/attribution-author.svg' );
+				background-repeat: no-repeat;
+				width: 14px;
+				height: 14px;
+				margin-right: 8px;
+			}
+
+			&-sa {
+				background-image: inline-svg( '../images/attribution-icon-share-a-like.svg' );
+				background-repeat: no-repeat;
+				width: 14px;
+				height: 14px;
+				margin-right: 8px;
+			}
+
+			&-fair {
+				background-image: inline-svg( '../images/attribution-license-generic-gnu-free.svg' );
+				background-repeat: no-repeat;
+				width: 14px;
+				height: 14px;
+				margin-right: 8px;
+			}
+
+			&-public {
+				background-image: inline-svg( '../images/attribution-license-public-domain.svg' );
+				background-repeat: no-repeat;
+				width: 14px;
+				height: 14px;
+				margin-right: 8px;
+			}
+
+			&-author {
+				width: 220px;
+				height: 14px;
+				margin-right: 4px;
+				font-family: 'Lato', 'Linux Libertine', 'Georgia', 'Times', serif;
+				font-size: 12px;
+				font-weight: normal;
```

Comment:
This may have been copy-pasted from zeplin... a whole bunch of `<something>: normal;` is usually not needed.